PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/IDEHelperImpl.java

```diff
@@ -503,10 +580,39 @@ public void saveAppServiceFile(@NotNull final AppServiceFileLegacy file, @NotNul
         AzureTaskManager.getInstance().runInModal(task);
     }
 
-    private void notifyDownloadSuccess(final AppServiceFileLegacy file, final File dest, final Project project) {
+    @SneakyThrows
+    @Override
+    public void saveAppServiceFile(@NotNull AppServiceFile file, @NotNull Object context, @Nullable File dest) {
```

Comment:
not a good idea to wrap project as context

Reply:
`IDEHelper` was defined in utils, where dependency for `Project` was not introduced.